.github/workflows/native_jni_s3_pytorch.yml

```diff
@@ -140,30 +140,26 @@ jobs:
       - name: Instal CUDA
         shell: cmd
         run: |
-          curl.exe -L http://developer.download.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.243_426.00_windows.exe -o cuda.exe
           curl.exe -L http://developer.download.nvidia.com/compute/cuda/10.2/Prod/local_installers/cuda_10.2.89_441.22_windows.exe -o cuda102.exe
           curl.exe -L https://developer.download.nvidia.com/compute/redist/cudnn/v7.6.4/cudnn-10.1-windows7-x64-v7.6.4.38.zip -o cudnn.zip
           cuda102.exe -s
-          cuda.exe -s
           mkdir cuda
           unzip.exe cudnn.zip
-          cp.exe -a cuda/include cuda/lib cuda/bin "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.1/"
           cp.exe -a cuda/include cuda/lib cuda/bin "C:/Program Files/NVIDIA GPU Computing Toolkit/CUDA/v10.2/"
           rm.exe -Rf cuda102.exe cuda.exe cudnn.zip cuda
       - name: Install CUDA 11
         shell: cmd
         run: |
-          curl.exe -L http://developer.download.nvidia.com/compute/cuda/11.0.2/local_installers/cuda_11.0.2_451.48_win10.exe -o cuda.exe
+          curl.exe -L https://developer.download.nvidia.com/compute/cuda/11.1.0/local_installers/cuda_11.1.0_456.43_win10.exe -o cuda.exe
```

Comment:
Just to confirm, have you verified the windows build here? Any link to the successful JNI build Action?

Reply:
not yet...can I take windows build in separate PR ?